custom_metrics/markup.js

```diff
@@ -560,7 +560,7 @@ try { // whole process is placed in a try/catch so we can log uncaught errors
         result.app_id_present = !!document.getElementById("app");
 
         // `<meta name="theme-color">` 
-        result.meta_theme_color = document.querySelector('meta[name="theme-color"]')?.getAttribute('content') ?? null;
+        result.meta_theme_color = document.querySelector('meta[name="theme-color"i]')?.getAttribute('content') ?? null;
```

Comment:
Nothing to do with this PR but this will only return the first one. Can't imagine why someone would want to specify multiple theme-colors but if they do, wouldn't the last one (not the first one) take effect?

Probably a real edge case, and can be totally ignored if you want (especially in terms of this PR), but since everything else uses querySelectorAll and I noticed this one used querySelector thought I'd ask.

Reply:
To be honest, this is only used to test if it is present or not. I probably passed back the value in case the chapter suddenly wanted it. 

Not sure how the recipient systems would deal with multiple entries. I know with Google and SEO things can vary per tag. First title, all meta descriptions, most restrictive robots tag etc.

Reply:
Grand. Resolving.